pysidt/sidt.py

```diff
@@ -1937,4 +1952,18 @@ def _assign_depths(node, depth=0):
 
 
 def is_prepruned_by_uncertainty(node):
-    return node.rule.uncertainty <= min(item.uncertainty for item in node.items)
\ No newline at end of file
+    return node.rule.uncertainty <= min(item.uncertainty for item in node.items)
+
+def get_bounded_fraction(errs, uncs, confidence_level):
+    t = scipy.stats.norm.ppf((1 + confidence_level) / 2)
+    return np.sum(uncs * t >= np.abs(errs)) / len(errs)
```

Comment:
I think this formula is wrong...the uncs you're pulling I believe are variances not standard deviations...also separately I think the len(errs) needs a sqrt?